pallets/aggregate/src/data.rs

```diff
@@ -209,13 +209,26 @@ pub enum AggregateSecurityRules {
 pub struct Delivery<B: sp_std::fmt::Debug + sp_std::cmp::PartialEq> {
     /// Destination
     pub destination: Destination,
-    /// Price
-    pub price: B,
+    /// fee
+    pub fee: B,
+    /// Tip for the delivery owner
+    pub owner_tip: B,
```

Comment:
Theoretically, this two fields can be merged into one no ? We could only keep the 'fee' param, which the delivery owner can set to include its tip as well

Reply:
This also avoids the need to handle the storage migration

Reply:
synced offline, we need to separate these 2 because only the fee is what we need to send to hyperbridge, since the top will remain on domain owner's account.